### PR TITLE
feat : customers review ui

### PR DESCRIPTION
**Pull request by afandi-siregar**
feat : https://3.basecamp.com/5190892/buckets/31702802/card_tables/cards/5889981429

### CODE REVIEW

File: api/restaurant/getReviews.ts [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+      sort: "newest",
+    });
+
+    const query = `${qs.stringify(payload)}`;
+    const { data, httpStatus, error } = await useHttp({
+      url: `reviews.json?${query}`,
+      method: "GET",
+    });
+
+    const result = responseReviewsSchema.parse({ data, httpStatus, error });
```
Comment (afif-hh) [inline api/restaurant/getReviews.ts:68]: yagn di parse cukup respond backend aja, coba liat di file" lain

File: components/restaurant_detail/restaurant_body/restaurant_review/ResturantCustomerReview.vue [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+    <CustomerReviewComment
+      :is-show-read-more="isShowReadMore"
+      :text-comment="textComment"
+      @show-more="showMoreComment"
+    />
+  </div>
+</template>
+
+<script lang="ts" setup>
+import "swiper/css";
```
Comment (afif-hh) [inline components/restaurant_detail/restaurant_body/restaurant_review/ResturantCustomerReview.vue:28]: remove swiper

File: section/customer_review/CustomerReviewGallery.vue [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+            class="rounded"
+          />
+        </component>
+      </component>
+    </template>
+  </div>
+</template>
+
+<script lang="ts" setup>
+import { ref, onMounted, PropType } from "vue";
```
Comment (afif-hh) [inline section/customer_review/CustomerReviewGallery.vue:29]: kurang import swiper css

**Review by afif-hh — CHANGES_REQUESTED** [review]
check again

**Comment by afandi-siregar** [comment]
done mas @afif-hh

**Pull request merged**